packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5815,6 +5816,10 @@ class RichText extends MultiChildRenderObjectWidget {
   /// How visual overflow should be handled.
   final TextOverflow overflow;
 
+  /// By default the TextOverflow.ellipsis shows "..."
+  /// Use [ellipsisValue] to show custom String
```

Comment:
See the comment above regarding documentation. You could use a dartdoc macro to reuse a piece of documentation (search for `@macro` and `@template` for examples).

Reply:
Same for the API documentation in text.dart.